src/index.json

```diff
@@ -951,6 +951,52 @@
                     "version": "0.9.0"
                 }
             }
+        ],
+        "loganalytics": [
+            {
+                "filename": "azure_loganalytics_cli_extension-1.0.0rc2-py2.py3-none-any.whl",
+                "sha256Digest": "38075c5df482053d5ca5a049b327616ba134c1e487fc79fb63904d8768ce9f63",
+                "downloadUrl": "https://files.pythonhosted.org/packages/4c/73/8a6a8327a2f64077bcfb905979ea2ebeb99e9d612602e01f5db9eb9f70ed/azure_loganalytics_cli_extension-1.0.0rc2-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.isPreview": true,
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 2",
+                        "Programming Language :: Python :: 2.7",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.4",
+                        "Programming Language :: Python :: 3.5",
+                        "Programming Language :: Python :: 3.6",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "aleldeib@microsoft.com",
+                                    "name": "Ace Eldeib",
+                                    "role": "author"
+                                }
+                            ],
+                            "document_names": {
+                              "description": "DESCRIPTION.rst"
+                            },
+                            "project_urls": {
+                                "Home": "https://github.com/Azure/azure-cli-extensions"
```

Comment:
You may want to point to "https://github.com/Azure/azure-cli-extensions/tree/master/src/log-analytics" so that people will be directed straight towards your readme.